src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py

```diff
@@ -163,41 +163,62 @@ def _get_calculated_data_document(
         return calculated_data_documents
 
     def _get_260_280(self, data: pd.DataFrame, row: int) -> CalculatedDataDocumentItem:
+        data_source_doc_items = []
+        if _get_str_or_none(data, row, "a260"):
+            data_source_doc_items.append(
+                DataSourceDocumentItem(
+                    data_source_feature="absorbance",
+                    data_source_identifier=_get_str(data, row, "a260 uuid"),
+                )
+            )
+
+        if _get_str_or_none(data, row, "a280") or _get_str_or_none(
+            data, row, "a280 10mm"
+        ):
+            data_source_doc_items.append(
+                DataSourceDocumentItem(
+                    data_source_feature="absorbance",
+                    data_source_identifier=_get_str(data, row, "a280 uuid"),
+                )
+            )
+
+        data_source_doc = None
+        if len(data_source_doc_items):
+            data_source_doc = DataSourceAggregateDocument(
+                data_source_document=data_source_doc_items
```

Comment:
Could you rename `data_source_doc` to `data_source_aggregated_document`? It's a bit confusing to read this as is.